templates/quickstart-hashicorp-consul.template

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: "HashiCorp Consul (Please do not remove) Aug,28,2019 (qs-1qup6ra8j)"
+Description: "HashiCorp Consul (Please do not remove) Aug,28,2019 QS(0036)"
```

Comment:
keep the QuickStart code (`qs-1qup6ra8j`) in the description

Reply:
This doesn't look like it is reverted. Please keep the (qs-1qup6ra8j) in the description.